acceptance-tests/tests/src/test/java/org/hyperledger/besu/tests/acceptance/multitenancy/MultiTenancyAcceptanceTest.java

```diff
@@ -0,0 +1,147 @@
+import org.hyperledger.besu.tests.acceptance.dsl.node.cluster.Cluster;
+import org.hyperledger.besu.tests.acceptance.dsl.node.cluster.ClusterConfiguration;
+import org.hyperledger.besu.tests.acceptance.dsl.node.cluster.ClusterConfigurationBuilder;
+
+import java.util.List;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.github.tomakehurst.wiremock.junit.WireMockRule;
+import org.junit.Before;
+import org.junit.Rule;
+import org.junit.Test;
+
+public class MultiTenancyAcceptanceTest extends AcceptanceTestBase {
+  private BesuNode node;
+  private Cluster multiTenancyCluster;
+  private String token;
+  private static final int ENCLAVE_PORT = 1080;
+  private final String privacyGroupId = "groupId";
+  private final String enclaveKey = "negmDcN2P4ODpqn/6WkJ02zT/0w0bjhGpkZ8UP6vARk=";
+
+  private final String key1 = "sgFkVOyFndZe/5SAZJO5UYbrl7pezHetveriBBWWnE8=";
+  private final String key2 = "R1kW75NQC9XX3kwNpyPjCBFflM29+XvnKKS9VLrUkzo=";
+  private final String key3 = "QzHuACXpfhoGAgrQriWJcDJ6MrUwcCvutKMoAn9KplQ=";
+
+  private final ObjectMapper mapper = new ObjectMapper();
+
+  @Rule public WireMockRule wireMockRule = new WireMockRule(options().port(ENCLAVE_PORT));
+
+  @Before
+  public void setUp() throws Exception {
+    final ClusterConfiguration clusterConfiguration =
+        new ClusterConfigurationBuilder().awaitPeerDiscovery(false).build();
+    multiTenancyCluster = new Cluster(clusterConfiguration, net);
+    node =
+        besu.createNodeWithMultiTenancyEnabled(
+            "node1",
+            "http://127.0.0.1:" + ENCLAVE_PORT,
+            "authentication/auth_priv.toml",
+            "authentication/auth_priv_key");
+    multiTenancyCluster.start(node);
+    token = node.execute(permissioningTransactions.createSuccessfulLogin("user", "pegasys"));
+  }
+
+  @Test
+  public void shouldGetPrivacyPrecompileAddress() {
+    final Integer precompileAddress = Integer.decode("0x7E");
+    node.useAuthenticationTokenInHeaderForJsonRpc(token);
+    node.verify(priv.privGetPrivacyPrecompileAddressSuccess(precompileAddress));
+  }
+
+  @Test
+  public void shouldGetPrivateTransaction() throws JsonProcessingException {
+    node.useAuthenticationTokenInHeaderForJsonRpc(token);
+
+    final String base64SignedPrivateTransactionRLP =
+        "+MyAAYJSCJQJXnuupqbHxMLf65d++sMmr1Uth6D//////////////////////////////////////////4AboEi1W/qRWseVxDGXjYpqmStijVV9pf91mzB9SVo2ZJNToB//0xCsdD83HeO59/nLVsCyitQ2AbSrlJ9T+qB70sgEoANWlbTMSwlB5gVR16Gc8wYD21v8I+WsQ6VvV/JfdUhqoA8gDohf8p6XPiV2tmABgdGworUpTjDZvkoZgf+zOguMinJlc3RyaWN0ZWQ=";
+
+    // privateFrom value from above transaction
+    final String privateFrom = "A1aVtMxLCUHmBVHXoZzzBgPbW/wj5axDpW9X8l91SGo=";
+
+    // Create a transaction
+    final Account sender = accounts.createAccount("account1");
+    final Hash transactionHash = node.execute(accountTransactions.createTransfer(sender, 50));
```

Comment:
These needs be a private marker transaction for this to work. This would be creating a transaction with an empty payload which works because you are responding for any reason on the enclave /receive